Sources/Kitura/Kitura.swift

```diff
@@ -155,10 +156,24 @@ public class Kitura {
     /// Kitura.addHTTPServer(onPort: 8080, with: router)
     /// Kitura.run()
     ///```
+    /// Make all registered servers start listening on their port and exit if any fail to start.
+    ///```swift
+    /// let router = Router()
+    /// Kitura.addHTTPServer(onPort: 8080, with: router)
+    /// Kitura.run(exitOnFailure: false)
```

Comment:
@ddunn2 I just spotted that the docs here are incorrect - since the default for `exitOnFailure` is `true`, the first example needs to set it to false, and the second example can use the default.  Or, swap the titles of the examples so that the first one (the default) is exit if there's a failure to start.